code/datums/components/blob_minion.dm

```diff
@@ -139,11 +139,17 @@
 /// We only speak telepathically to blobs
 /datum/component/blob_minion/proc/on_try_speech(mob/living/minion, message, ignore_spam, forced)
 	SIGNAL_HANDLER
-	minion.log_talk(message, LOG_SAY, tag = "blob hivemind telepathy")
-	var/spanned_message = minion.say_quote(message)
+	// Invoke async to avoid stopping.
```

Comment:
```suggestion
```

i think it's fairly obvious why we're invoking async in a `SIGNAL_HANDLER` proc. if you want to have a useful bit about sleeping then add a comment on the line that sleeps in `send_blob_telepathy()`